.circleci/config.yml

```diff
@@ -119,41 +119,40 @@ jobs:
         type: enum
         enum: ["production", "staging", "development", "e2e", "raiden-package"]
     executor: base-executor
-    working_directory: << pipeline.parameters.working_directory >>
+    working_directory: << pipeline.parameters.working_directory >>/raiden-dapp
     steps:
       - attach_workspace: *attach_options
       - run:
           name: Build dapp
-          command: pnpm run build --filter raiden-dapp -- --mode << parameters.mode >>
+          command: pnpm run build -- --mode << parameters.mode >>
       - run:
           name: Compress dapp
-          working_directory: raiden-dapp
           command: tar -czvf dapp.tar.gz dist
```

Comment:
How can it be that we don't have to adjust the command when we change the working dir?

Reply:
I was changed (checkout [this version](https://github.com/raiden-network/light-client/blob/447a642baadd42b9503288f707d6cf584971a1bd/.circleci/config.yml#L194). GitHub is just pretty bad in showing a good diff here since the order of the job definition has changed.